core/src/entry.rs

```diff
@@ -557,8 +553,8 @@ mod tests {
         let tx = system_transaction::create_user_account(&keypair, &keypair.pubkey(), 0, zero, 0);
         let entry = next_entry(&zero, 1, vec![tx.clone()]);
         assert_eq!(
-            Entry::serialized_size(&[tx]),
-            serialized_size(&entry).unwrap()
+            Entry::to_blob_size(&[tx]),
+            serialized_size(&vec![entry]).unwrap() // blobs are Vec<Entry>
```

Comment:
this was the test that failed once line 557 was updated to reflect what to_blobs() does